book/src/build_wasm.md

```diff
@@ -0,0 +1,74 @@
+# Building a WASM binary manually
+
+Here are the complete commands for building a WASM binary with this library:
+
+```bash
+export RUSTUP_TOOLCHAIN=nightly
+export RUSTFLAGS="-C target-feature=+atomics,+bulk-memory,+mutable-globals"
+wasm-pack build \
+    -t no-modules \
+    -d <WASM_OUTPUT_PATH> \
+    --no-typescript -- \
+    -Z build-std=std,panic_abort
+```
+
+Continue reading for more details.
+
+---
+
+`flutter_rust_bridge_codegen` expects a certain setup that is modeled after the
+[wasm_bindgen raytracing example](https://github.com/rustwasm/wasm-bindgen/tree/main/examples/raytrace-parallel)
+and by extension consumes the [`wasm_bindgen`] library and its ecosystem.
+The requirements are:
+- The standard library being built with the `panic_abort` feature
```

Comment:
nit: I like panic unwind... Is it possible for us to use unwind? (may have commented somewhere before, and this is just a wish not a requirement)

Reply:
Might be hard. As of now Rust refuses to compile without the feature, and WASM exception handling is work-in-progress on all major browsers according to the [roadmap](https://webassembly.org/roadmap/). To put it in another way, we are at the cutting edge of process 🎉